src/Altinn.AccessManagement.Core/Services/MaskinportenSchemaService.cs

```diff
@@ -43,6 +42,63 @@ public MaskinportenSchemaService(ILogger<IMaskinportenSchemaService> logger, IDe
             _pap = pap;
         }
 
+        /// <inheritdoc/>
+        public async Task<DelegationCheckResult> DelegationCheck(int authenticatedUserId, int authenticatedUserAuthlevel, RightsDelegationCheckRequest request)
+        {
+            (DelegationCheckResult result, ServiceResource resource, Party fromParty) = await ValidateDelegationCheckRequest(request);
+            if (!result.IsValid)
+            {
+                return result;
+            }
+
+            DelegationHelper.TryGetResourceFromAttributeMatch(request.Resource, out ResourceAttributeMatchType resourceMatchType, out string resourceRegistryId, out string org, out string app, out string serviceCode, out string serviceEditionCode);
```

Comment:
org, app, serviceCode and serviceEditionCode out params will never be used here and can be omitted by setting them to _